src/map.rs

```diff
@@ -68,14 +68,18 @@ impl<K: Eq + Hash, V> FrozenMap<K, V> {
 }
 
 impl<K: Eq + Hash, V: StableDeref, S: BuildHasher> FrozenMap<K, V, S> {
+    pub fn insert(&self, k: K, v: V) -> &V::Target {
+        self.insert_with(k, || v)
+    }
+
     // these should never return &K or &V
     // these should never delete any entries
-    pub fn insert(&self, k: K, v: V) -> &V::Target {
+    pub fn insert_with(&self, k: K, v: impl FnOnce() -> V) -> &V::Target {
```

Comment:
please update the safety comment on `in_use` to mention the `insert_with` callback